src/components/picker/types.tsx

```diff
@@ -59,6 +131,47 @@ export type PickerBaseProps = Omit<NewTextFieldProps, 'value' | 'onChange'> & {
    * Temporary prop required for migration to Picker's new API
    */
   migrate?: boolean;
+  /**
+   * @deprecated
+   * A function that extract the unique value out of the value prop in case value has a custom structure (e.g. {myValue, myLabel})
+   */
+  getItemValue?: (value: PickerValue) => any;
+  /**
+   * @deprecated
+   * A function that extract the label out of the value prop in case value has a custom structure (e.g. {myValue, myLabel})
+   */
+  getItemLabel?: (value: PickerValue) => string;
+  /**
+   * @deprecated
+   * Callback for modal onShow event
+   */
+  onShow?: () => void;
+  /**
+   * @deprecated
+   */
+  children?: ReactNode | undefined;
+  /**
+   * @deprecated
+   * The picker modal top bar props
+   */
+  topBarProps?: ModalTopBarProps;
+  /**
+   * @deprecated
+   * Render custom picker modal (e.g ({visible, children, toggleModal}) => {...})
+   */
+  renderCustomModal?: (modalProps: RenderCustomModalProps) => React.ReactElement;
+  /**
+   * @deprecated
+   * Pass props to the picker modal
+   */
+  pickerModalProps?: object;
+};
+
+export type PickerTypes = PickerTypeProps | PickerPropsDeprecation;
```

Comment:
Where do you use 'PickerTypes'?

Reply:
I think you can remove this and rename 'PickerModeTypes' -> 'PickerTypes'/'PickerOverlayTypes"/'PickerExpandableTypes" etc... 
The Picker "mode" refers to single/multi and there's already a "PickerModes" type for that. 
It can be confusing...

Reply:
Actually I'll remove it (`PickerTypes`), there is not a use in this type.
I used it before for the conditional typings I think.

I'll rename `PickerModeTypes ` - `PickerTypes`